lib/rubocop/cop/rails/find_by.rb

```diff
@@ -59,8 +59,7 @@ def autocorrect(corrector, node)
           return if node.method?(:first)
 
           where_loc = node.receiver.loc.selector
-          first_loc = range_between(node.loc.dot.begin_pos, node.loc.selector.end_pos)
-
+          first_loc = range_between(node.receiver.source_range.end_pos, node.loc.selector.end_pos)
```

Comment:
Can you keep the intentional blank line?

```suggestion
          first_loc = range_between(node.receiver.source_range.end_pos, node.loc.selector.end_pos)

```

Reply:
Thanks! I've fixed.